src/hw-monitor.h

```diff
@@ -266,7 +119,12 @@ namespace librealsense
         }
     };
 
-    std::string hwmon_error_string( command const &, hwmon_response e );
+    typedef int32_t hwmon_response;
+    class base_hwmon_response_handler { // base class for different product number to implement responses
```

Comment:
On most places we name interfaces as `XXX_interface`. Consider renaming `hwmon_response_handler_interface` or `hwmon_response_interface` (are we a "handler"?).
Also can wrap `typedef int32_t hwmon_response` in the class namespace as `hwmon_response_interface::type`